Modix.Bot/Modules/DocumentationModule.cs

```diff
@@ -24,36 +27,49 @@ public DocumentationModule(DocumentationService documentationService)
             [Summary("The term to search for in the documentation.")]
                 string term)
         {
+
+            Regex reg = new Regex("^[0-9A-Za-z.<>]+$");
+            foreach(char c in term)
+            {
+                if(!reg.IsMatch(c.ToString()))
+                {
+                    await ReplyAsync($" ' {c} ' character is not allowed in the search, Please try again.");
```

Comment:
Nit: I'm a bit nitpicky on grammar 😛. Should probably either be `$"'{c}' character is not allowed in the search. Please try again."` or `$"'{c}' character is not allowed in the search, please try again."`